frontend/lib/pages/base/home/home_page.dart

```diff
@@ -10,9 +11,27 @@ class HomePage extends StatefulWidget {
 
 class _HomePageState extends State<HomePage> {
   @override
-  Widget build(BuildContext context) {
-    return Container(
-      color: spqWhite,
-    );
-  }
-}
+  Widget build(BuildContext context) => Scaffold(
+
+    appBar: AppBar(
+      backgroundColor: Colors.white,
+      elevation: 4.0,
+
+      leading: Builder(
+        builder: (context) {
```

Comment:
Do you really need this Builder here? Can't you just use the build context form the build method?